.github/workflows/ci.yml

```diff
@@ -534,49 +516,14 @@ jobs:
 
 # Android jobs
 
-  build-android:
-    runs-on: ubuntu-latest
-    name: Build Android
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-        with:
-          submodules: 'recursive'
-
-      - name: Check cache
-        id: check-cache
-        if: ${{ !contains(github.head_ref, 'release/') }}
-        uses: actions/cache@v3
-        with:
-          path: ./binary/android/**
-          key: binaries-android-${{hashFiles('./src/**')}}
-
-      - name: Setup Ninja
-        if: contains(github.head_ref, 'release/') || steps.check-cache.outputs.cache-hit != 'true'
-        uses: seanmiddleditch/gha-setup-ninja@master
-
-      - name: Build Realm for Android
-        if: ${{ contains(github.head_ref, 'release/') || steps.check-cache.outputs.cache-hit != 'true' }}
-        # TODO: parallelize build for all architectures
-        run: |
-          export ANDROID_NDK=$ANDROID_NDK_HOME
-          ./scripts/build-android.sh
-
-      - name: Store artifacts
-        uses: actions/upload-artifact@v2
-        with:
-          name: librealm-android
-          path: binary/android/**
-          retention-days: 1
-
   flutter-android:
```

Comment:
Just an idea: I think, in order to be sure that the build is correct for all the architectures we will have to `Create Android Emulator` and `Run tests on Android Emulator` for all the architectures. It also could be done in a matrix.
The same for IOS. We could do it for the master only.